libraries/abstractions/platform/test/iot_test_platform_threads.c

```diff
@@ -186,23 +170,7 @@ TEST( UTIL_Platform_Threads, IotThreads_CreateDetachedThread )
  */
     void threadStackSizeTestFunction( void * param )
     {
-        TaskStatus_t xTaskDetails;
-
-        /* Use the handle to obtain further information about the task. */
-        vTaskGetInfo( /* The handle of the task being queried. */
-            NULL,
-
-            /* The TaskStatus_t structure to complete with information
-             * on xTask. */
-            &xTaskDetails,
-
-            /* Include the stack high water mark value in the
-             * TaskStatus_t structure. */
-            pdTRUE,
-            /* Include the task state in the TaskStatus_t structure. */
-            eInvalid );
-
-        *( int32_t * ) param = 300 + xTaskDetails.usStackHighWaterMark;
+        *( int32_t * ) param = 300 + uxTaskGetStackHighWaterMark( NULL );
```

Comment:
same as comment  `uxTaskPriorityGet()`

Reply:
I went ahead and removed this function. It's unused.